pkg/oc/admin/migrate/scc/helpers.go

```diff
@@ -0,0 +1,46 @@
+package scc
+
+import (
+	"fmt"
+
+	"k8s.io/api/core/v1"
+	extensions "k8s.io/api/extensions/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+)
+
+func convertCapabilities(inputCapabilities []kapi.Capability) []v1.Capability {
```

Comment:
Any particular reason not to use `Convert_core_Capabilities_To_v1_Capabilities` ?

Reply:
Yes, because `Capabilities` is a struct but here we convert a list of `Capability` (they are just a strings in fact).

Reply:
Tangibly ok.